homeassistant/components/climate/mysensors.py

```diff
@@ -0,0 +1,216 @@
+"""
+mysensors platform that offers a Climate(MySensors-HVAC) component.
+
+For more details about this platform, please refer to the documentation
+https://home-assistant.io/components/climate.mysensors
+
+Curently supported operations:
+
+set_temperature()    --> V_HVAC_SETPOINT_COOL, V_HVAC_SETPOINT_HEAT
+set_operation_mode() --> V_HVAC_FLOW_STATE
+set_fan_mode()       --> V_HVAC_SPEED
+
+Pending:
+humidity, away_mode, aux_heat, swing_mode
+
+home-assistant/components/climate/mysensors.py
```

Comment:
I think you can move the details about supported operations to the docs at home-assistant.io. Just keep the header and link to the doc page as is custom for all platforms. Make sure you explain in the docs, that the user should either make a device with one setpoint or a device that has two setpoints. If the device only has one setpoint, the user should decide which one of V_HVAC_SETPOINT_HEAT or V_HVAC_SETPOINT_COOL to use.